### PR TITLE
PT-8810 - 5.4 compatibility

### PR DESCRIPTION
**Pull request by swDennis**

### CODE REVIEW

File: exampleplugins/5.2/SeoExample/SeoExample.php [hunk 1]
```diff
@@ -79,7 +79,9 @@ public function loadPerformanceExtension(\Enlight_Controller_ActionEventArgs $ar
             return;
         }
 
-        $subject->View()->addTemplateDir(__DIR__ . '/Resources/views/');
+        $pluginBasePath = $this->container->getParameter('seo_example.plugin_dir');
+
+        $subject->View()->addTemplateDir($pluginBasePath . '/Resources/views/');
```
Comment (mitelg) [inline exampleplugins/5.2/SeoExample/SeoExample.php]: as you are in the base plugin class, you can use `$this->getPath()`
no need for getting the container parameter

File: exampleplugins/5.2/SwagAttribute/SwagAttribute.php [hunk 2]
```diff
@@ -42,13 +42,12 @@ public function install(InstallContext $context)
             'custom' => true,
 
             //in case of combo box type, defines the selectable values
-            'arrayStore' => [
+            'store' => [
```
Comment (mitelg) [inline exampleplugins/5.2/SwagAttribute/SwagAttribute.php]: are you sure this works? the column in s_attribute_configuration is called array_store

File: source/developers-guide/attribute-system/index.md [hunk 3]
```diff
@@ -189,7 +189,7 @@ class SwagAttribute extends Plugin
             'custom' => true,
 
             //in case of combo box type, defines the selectable values
-            'arrayStore' => [
+            'store' => [
```
Comment (mitelg) [inline source/developers-guide/attribute-system/index.md]: same as above

File: exampleplugins/5.2/SwagAttribute/Resources/views/backend/swag_attribute/SwagAttribute.FieldHandler.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        return null;
+    },
+
+    createOwnTypeField: function(field) {
+        return Ext.apply(field, {
+            xtype: 'swag-attribute-type'
+        });
+    },
+
+    createOwnValidationField: function(field) {
```
Comment (buddhaCode) [inline exampleplugins/5.2/SwagAttribute/Resources/views/backend/swag_attribute/SwagAttribute.FieldHandler.js]: can't wait to see this featrue. wooohoo.

File: exampleplugins/5.2/SeoExample/SeoExample.php (hunk 1 above)
Comment (swDennis) [reply exampleplugins/5.2/SeoExample/SeoExample.php]: sure. my fault

**Pull request merged**